src/main/scala/se/nimsa/sbx/util/FutureUtil.scala

```diff
@@ -16,21 +16,130 @@
 
```

Comment:
In this file, a few versions of retry for Futures are added. They ended up not being used but they are left here for now. May clean out these after finishing the next PR for poll transfers.